package.json

```diff
@@ -2,25 +2,89 @@
   "name": "hill-chart",
   "description": "Basecamp's hill chart implementation in d3.js",
   "version": "1.3.1",
+  "author": "Ahmed Nagi <ahmedflnagi@gmail.com> (https://ahmednagi.com)",
+  "license": "MIT",
   "main": "dist/hill-chart.cjs.js",
   "module": "dist/hill-chart.esm.js",
   "style": "dist/styles.css",
   "files": [
     "dist"
   ],
+  "engines": {
+    "node": ">=14"
+  },
```

Comment:
Technically it works, but not sure if we should increase this even further to `16` because of https://endoflife.date/nodejs.

Reply:
node 14 is a famous version alot of projects still depend on it so I guess we stick to it for now.